Essentials/src/main/java/net/essentialsx/api/v2/services/EssentialsService.java

```diff
@@ -0,0 +1,7 @@
+package net.essentialsx.api.v2.services;
+
+/**
+ * An EssentialsX service that provides basic information about itself.
```

Comment:
```suggestion
 * A service provided by EssentialsX.
```
Not sure if we actually need to keep this interface at all? (Yes I probably should've brought that up before, don't judge me)